packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -302,12 +305,15 @@ Generated by [AVA](https://ava.li).
       A Netlify Function failed to require one of its dependencies.␊
       Please make sure it is present in the site's top-level "package.json".␊
     ␊
-      In file "/file/path"␊
+      In file "packages/build/tests/install/fixtures/mispelled_dep/functions/index.js"␊
+    ␊
+      Require stack:␊
+     REQUIRE STACK␊
```

Comment:
From what I reviewed in the regular expressions, this is expected 👍

Reply:
Correct!